test/integration/tests/fields/standalone.js

```diff
@@ -0,0 +1,54 @@
+/* @flow */
```

Comment:
I think this file is not being imported right now. Is the test actually running?